omnipaxos/src/messages.rs

```diff
@@ -224,7 +227,9 @@ pub mod ballot_leader_election {
         /// Number of the current heartbeat round.
         pub round: u32,
         /// BLEBallot of a replica.
-        pub ballot: BLEBallot,
+        pub ballot: Ballot,
+        /// The number of replicas inside the cluster the sender is connected to (including itself)
+        pub connectivity: Connectivity,
```

Comment:
I think the connectivity alias is mainly to make the BLE code more readable so I prefer not to expose it publicly.
```suggestion
        pub connectivity: u8,
```

Reply:
Yes